### DISCUSSION TITLE
Any idea on how to flash on ASUS motherboards with no Flashback feature?

### DISCUSSION THREAD

**Discussion by BanefulDemon** [xCuri0/ReBarUEFI#145] [Q&A]
I'm using a Z390-F and EZFlash won't let me flash, says security verification failed and my mobo doesn't have the USB Bios Flashback feature. Any help would be appreciated.

**Comment by MrAnonymous7122** [comment]
You need to buy a CH341A Programmer, they're very cheap and work well.
